src/main/java/edu/harvard/iq/dataverse/util/SystemConfig.java

```diff
@@ -1112,7 +1112,8 @@ public boolean isDatafileValidationOnPublishEnabled() {
     }
 
 	public boolean directUploadEnabled(DvObjectContainer container) {
-    	return Boolean.getBoolean("dataverse.files." + container.getEffectiveStorageDriverId() + ".upload-redirect");
+        // this method is used in UI only, therfore "dataverse.files." + driverId + ".allow-out-of-band-upload" is not used here
+    	return Boolean.parseBoolean(System.getProperty("dataverse.files." + container.getEffectiveStorageDriverId() + ".upload-redirect"));
```

Comment:
Could you fix the indentation, too? Thx!

Reply:
This was by accident (I did search for all occurrences of the relevant jvm option and found this file and realized to late it was old release notes). I have reverted it.

Reply:
No problem!